### ISSUE TITLE
[BUG] from build 005684d8 and a little older

### ISSUE THREAD

**Issue by Shadow322** [LumaTeam/Luma3DS#601]
when having payloads on nand only on both O3DS/N3DS chainloader does not boot any payloads and just boots like there is none to 3ds menu

**Comment by TuxSH** [comment]
I don't understand your sentence, could you rephrase into a well-formed one, please ?
EDIT: thank you.

**Comment by AuroraWright** [comment]
Can't reproduce...

**Comment by Shadow322** [comment]
OK i fixed it deleted both code.bin from sd and nand and made a new one in nand and now it works so might be something to look into
